src/ld65/exports.c

```diff
@@ -99,6 +99,114 @@ static Export* NewExport (unsigned Type, unsigned char AddrSize,
 
 
 
+static void UpdateExportsInImportList (Export *E)
+/* Update all exports in the given export's import list. */
```

Comment:
The predominant style is that those short "statements" don't end with a period. (There are several instances of this in the PR.)